### PR TITLE
Require verified Tella editor fixture leases

### PR DESCRIPTION
**Pull request by kentdebruin**
## Summary
- replace self-reported editor fixture evidence with server-side `tella-stage` verification
- require and consistently propagate the creator's verified-login personal OAuth identity
- block Tella editor routes from the generic Portal setter and require the exact authoritative route shape
- derive the video, path, expiry, and reservation key from Tella's lease
- reject invented, expired, mismatched, inaccessible, and malformed fixture responses

Depends on merged and deployed tella-fusion#6104.
Supersedes #268 after rebasing onto current main; its current-head review approved, while unrelated CI infrastructure jobs failed.

## Verification
- `bun run check`
- focused Portal, MCP auth, creator identity, lease, opening-run policy, instruction, and catalog tests

Started by Kent de Bruin in [this OS session](https://os.tella.dev/session/os-01a057a7-74e5-7b22-86d1-fcc0e4448bc8)

### CODE REVIEW

File: packages/core/opensession-server/src/server/portals-mcp.ts [hunk 1]
```diff
@@ -449,6 +459,10 @@ export function createPortalsMcpServer(ctx: PortalsMcpContext) {
           const dir = workspace(ctx);
           if (dir instanceof Error) return result(dir.message);
           try {
+            if (isTellaEditorPath(path))
```
[the remark this answers is not in the document]
Comment (kentdebruin) [reply packages/core/opensession-server/src/server/portals-mcp.ts:462]: Fixed in a9bbd592a on #285: generic paths are now canonicalized through `URL.pathname` before editor-route classification, with a regression test for `/other/../video/<id>/edit`.

**Comment by kentdebruin** [comment]
Superseded by #285, rebased onto current main with the current-head canonical-path finding fixed. CI failures were also investigated: all 685 isolated tests report zero failures before xargs exits 123, and the Ubuntu startup timeout reproduces on current main.
